result/rsvp.yaml

```diff
@@ -0,0 +1,192 @@
+openapi: 3.1.0
+
+info:
+  title: RSVP-TE metric results model.
+  version: ^0.0.0
+
+components:
+  schemas:
+    Rsvp.Metrics.Request:
+      description: >-
+        The request to retrieve RSVP-TE per Router metrics/statistics.
+      type: object
+      properties:
+        router_names:
+          description: >-
+            The names of RSVP-TE Routers to return results for.
+            An empty list will return results for all RSVP-TE routers.
```

Comment:
An empty list as input will return results for all RSVP-TE routers.

Reply:
Done.

Reply:
Done.